iam/api/Access/AddMember.cs

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// [START iam_modify_policy_member]
```

Comment:
The VIEW ON GITHUB links on this page:
https://cloud.google.com/iam/docs/granting-changing-revoking-access#iam-get-policy-csharp
Will all break if you submit this PR now.  I'll e-mail you and describe how to avoid breaking them.

Reply:
Done.